### DISCUSSION TITLE
Install all components at once

### DISCUSSION THREAD

**Discussion by andrecasal** [radix-ui/primitives#2510] [Help]
Is it possible to install all components at once instead of having to run `npm i @radix-ui/<component-name>` for every single one, and having ~20 entries in package.json?

![Screenshot 2023-11-08 at 08 56 52](https://github.com/radix-ui/primitives/assets/22457463/dd3340ee-f650-40ff-bbb6-ed1b514e091d)

**Comment by benoitgrelard** [comment]
This isn't something that is possible currently as each component is an independent package for incremental adoption purpose.
